src/components/Preview.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Container, Grid, Image } from "semantic-ui-react";
+import "../css/Preview.css";
+
+const Preview = ({ selectedArticle }) => {
+  return (
+    <Container id="preview" align="center">
+      <Grid>
+        <Grid.Row centered>
+          <Container id={"image"}>
```

Comment:
why is the id in { } ?

Reply:
good question..?!

Reply:
Copied grid structure from singlearticle in client, it had string interpolation there, which I remove the dynamic part of. Auto formatter changed into normal string, forgot to remove {}.
Addressing this